packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -353,8 +364,10 @@ export class MonacoEditorProvider {
     protected installReferencesController(editor: MonacoEditor): void {
```

Comment:
Maybe we don't need it anymore, It seems that this code was fixed in VS Code: https://github.com/spahnke/vscode/commit/b9f4d37650e21167eb04747e8b380598d6f9f5dd

Let's try to remove it and test that peek widget is not closed while changing editors.

Reply:
I tried to remove it and found that it doesn't work correctly. It works when I open a first editor, but the dialog is closed when I try to open a next editor.

![peek_reference_bug](https://user-images.githubusercontent.com/5676062/75981080-66e81500-5eec-11ea-9dc3-69b80c876784.gif)

So it's still in the PR `TODO` list  as unresolved. I will investigate why it works for `vs code`, but doesn't work for `theia`.

Reply:
Tested and debugged against https://github.com/theia-ide/vscode/tree/standalone/0.19.x.
`referencesController` [works well](https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts#L231) tor this case.
Investigating `theia` side...

Reply:
For case when `peek` widget is open VS Code doesn't create `codeEditorWidget` every time an editor opens. As result [the condition](https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts#L229) is met and `peek` widget is open.
Theia creates  `standaloneCodeEditor` instance every time, so the condition [is NOT met](https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts#L234), so `peek` widget on the next iteration [is closed](https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts#L225).
I think that's why we have this problem for `Theia` and don't have for `VS Code`

Reply:
ok, let's keep our fix